whois-commons/src/main/java/net/ripe/db/whois/common/dao/EmailStatusDao.java

```diff
@@ -26,6 +32,19 @@ public void createEmailStatus(final String email, final EmailStatus emailStatus)
                 LocalDateTime.now());
     }
 
+    public Map<String, String> getEmailStatus(final Set<String> emailAddresses) {
+        Map<String, String> results = Maps.newHashMap();
+
+        namedParameterJdbcTemplate.query(
+                "SELECT email, status from email_status where email in (:emails)",
```

Comment:
Correct formatting? FROM, IN and WHERE should also be in caps?

Reply:
it should be in uppercase, because they are keywords, you are right.
In the code we have a mix.